src/encoded/tests/data/master-inserts/page.json

```diff
@@ -373,6 +374,24 @@
             "9626d82e-8110-4213-ac75-0a50adf890ff"
         ]
     },
+    {
+        "identifier": "docs/user-guide/downloading-files",
+        "uuid": "3783757d-cc48-49a0-a76e-fc26ed462a24",
+        "title": "How to Download Files",
+        "content": [
+            "1b656dca-547b-4c2a-b61b-57d4903b4599"
+        ],
+        "table-of-contents": {
+            "enabled": true
+        },
+        "status": "public",
+        "consortia": [
+            "358aed10-9b9d-4e26-ab84-4bd162da182b"
+        ],
+        "submission_centers": [
+            "9626d82e-8110-4213-ac75-0a50adf890ff"
+        ]
+    },
```

Comment:
Just a heads up, there was already a page available for this `859b2c48-0bda-406c-b49b-07d68bd236d4`. If we're going with this one, please delete the other one from this file.